scss/jetpack-wizard-banner.scss

```
@@ -0,0 +1,110 @@
+@import '../_inc/client/scss/mixin_breakpoint';
+@import '../_inc/client/scss/calypso-colors';
+
+
+.jp-wizard-banner {
+	text-align: left;
+	background: $white;
+	padding: 2rem;
+	box-shadow: 0px 2px 6px rgba( 0, 0, 0, 0.03 ), 0px 1px 2px rgba( 0, 0, 0, 0.03 );
+	border: 1px solid #dcdcde;
```

I wonder if we should move away from Calypso colors and use Color studio instead; that will give us access to more up to date colors and more variables:

```suggestion
@import 'node_modules/@automattic/color-studio/dist/color-variables.scss';
@import '../_inc/client/scss/mixin_breakpoint';


.jp-wizard-banner {
	text-align: left;
	background: $studio-white;
	padding: 2rem;
	box-shadow: 0px 2px 6px rgba( 0, 0, 0, 0.03 ), 0px 1px 2px rgba( 0, 0, 0, 0.03 );
	border: 1px solid $studio-gray-5;
```